frontend/src/Elements/Elements.js

```diff
@@ -1,24 +1,6 @@
 import Element from './Element';
-import { CG } from '../js-sdk';
-import config from '../config';
 
 class Elements {
-  /**
-   * Set up the CG SDK.
-   * Inject CG to `window` if is not declared to be accessed by components.
-   */
-  constructor({ apiKey, apiUrl }) {
```

Comment:
In current state `Elements` component is redundant and I could remove it at all. But, probably it will be useful for further development so I decided not to do this.

Reply:
Okey, makes sense, lets keep it 👍